packages/account/src/Components/poi/status/unsupported/unsupported.tsx

```diff
@@ -49,25 +53,75 @@ const Unsupported = ({
     allow_poi_resubmission,
     handleViewComplete,
     onfido,
+    routeBackTo,
+    app_routing_history = [],
     ...props
-}: Partial<TUnsupported>) => {
+}: TUnsupported) => {
     const [detail, setDetail] = React.useState<number | null>(null);
+
     const toggleDetail = (index: number) => setDetail(index);
 
     const documents = getDocumentIndex({
         country_code,
     });
 
+    const from_platform = getPlatformRedirect(app_routing_history);
+
+    const onClickRedirectButton = () => {
+        const platform = platforms[from_platform?.ref as keyof typeof platforms];
+        const { is_hard_redirect = false, url = '' } = platform ?? {};
+        if (is_hard_redirect) {
+            window.location.href = url;
```

Comment:
Is there a specific case to set href directly from the component? Can't we use react router or useHistory?

Reply:
I just left previous implementation. so no changes here as it is not related to the PR.
we can check while refactoring manual upload feature